Lesson02-GitHub-Repo-Search/T02.06-Solution-AddPolish/app/src/main/java/com/example/android/datafrominternet/MainActivity.java

```diff
@@ -42,7 +42,7 @@
     private TextView mErrorMessageDisplay;
 
     // COMPLETED (20) Create a ProgressBar variable to store a reference to the ProgressBar
-    private ProgressBar mLoadingIndicator;
+    private ProgressBar mProgressBar;
```

Comment:
Just the variable name, it was inconsistent with the rest of the changes I've made.